src/app/content/components/Seats.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Flex, Heading, Progress } from '@chakra-ui/react';
+import { Seat } from '../../../fetchers';
+
+export interface SeatsProps {
+  seat?: Seat;
+}
+
+export function SeatInfo({ seat }: SeatsProps): JSX.Element {
+  const seatsPercent = seat === undefined ? undefined : (seat.seats.actual / seat.seats.capacity) * 100;
+  const waitlistPercent =
+    seat === undefined ? undefined : (seat.waitListSeats.actual / seat.waitListSeats.capacity) * 100;
+  return (
+    <Flex my="2" direction={{ base: 'column', md: 'row' }}>
+      <Box width={{ base: '100%', md: '50%' }}>
+        <Flex justifyContent="space-between" my="1">
+          <Heading as="h6" size="sm">
+            Seats
+          </Heading>
+          <Heading as="h6" size="sm">
+            {seat === undefined ? '' : `${seat.seats.actual} / ${seat.seats.capacity}`}
```

Comment:
im confused on what this line is doing?

Reply:
In response to your comment about showing how many seats are remaining, that is what this is doing. It's using what's called a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator), which is basically an if statement on one line, to form the string. It'll be like `10/100` or whatever.

Reply:
yeah i get that! i just remembered that seating backedn was broken thats why the seats display wasnt working haha